pkg/loadbalancerservice/controller.go

```diff
@@ -20,23 +22,38 @@ import (
 	"github.com/openshift/microshift/pkg/servicemanager"
 )
 
-const defaultInformerResyncPeriod = 10 * time.Minute
+const (
+	defaultInformerResyncPeriod = 10 * time.Minute
+)
 
 type LoadbalancerServiceController struct {
-	NodeIP     string
-	KubeConfig string
-	client     *kubernetes.Clientset
-	indexer    cache.Indexer
-	queue      workqueue.RateLimitingInterface
-	informer   cache.SharedIndexInformer
+	IPAddresses []string
+	NICNames    []string
+	NodeIP      string
+	KubeConfig  string
+	client      *kubernetes.Clientset
+	indexer     cache.Indexer
+	queue       workqueue.RateLimitingInterface
+	informer    cache.SharedIndexInformer
 }
 
 var _ servicemanager.Service = &LoadbalancerServiceController{}
 
 func NewLoadbalancerServiceController(cfg *config.Config) *LoadbalancerServiceController {
+	ipAddresses := make([]string, 0)
+	nicNames := make([]string, 0)
```

Comment:
It's no like leaving it at 0 will give use slices without any wasted space after all that appending so we could just do it ahead of time
```suggestion
	ipAddresses := make([]string, 0, len(cfg.Ingress.ListenAddress))
	nicNames := make([]string, 0, len(cfg.Ingress.ListenAddress))
```

Reply:
Same comment as above